workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -0,0 +1,77 @@
+using System.IO;
+using System.Text;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Tools;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public static class LaunchMenu
+    {
+        private const string rootApkPath = "build";
+        private const string MenuLaunchAndroid = "SpatialOS/Launch mobile client/Android Device";
+
+        [MenuItem(MenuLaunchAndroid, false, 73)]
+        private static void LaunchMobileClient()
+        {
+            try
+            {
+                EditorUtility.DisplayProgressBar("Launching Mobile Client", "Installing APK", 0.3f);
+
+                // Find apk to install
+                var rootPath = Path.Combine(Application.dataPath, Path.Combine("..", rootApkPath));
```

Comment:
that could be factored out into a static string, right? we need it for both menu items

Reply:
It can't, `Application.dataPath` can only be set on runtime.
I could make it a static property however?

Reply:
sure